Formula/restview.rb

```diff
@@ -63,7 +63,7 @@ class Restview < Formula
   end
 
   def install
-    venv = virtualenv_create(libexec, "python3.8")
+    venv = virtualenv_create(libexec, "python3.9")
```

Comment:
Could we do 
```suggestion
    venv = virtualenv_create(libexec, "python#{version.major_minor}")
```

Reply:
Some formulas have

```
venv = virtualenv_create(libexec, "python3")
```

others have

```
venv = virtualenv_create(libexec, Formula["python@3.8"].bin/"python3")
```

and few have

```
venv = virtualenv_create(libexec, "python3.8")
```

It's not clear to me why and what the differences are.

Reply:
@iMichka you worked on this before, any idea? Could this be a rubocop?

Reply:
I think that python3.8 / python3.8 / python3 means: whatever python is first in the path. So any anaconda, python.org or anything else.

I had a preference for Formula["python@3.8"].bin/"python3", as this was the Python we where shipping.

But we need to double check that information. And yes a rubocop is probably needed here to be consistent.

Reply:
May as well add a check that ensures the correct `python@3.x` formula is mentioned everywhere. So changing the `depends_on "python@3.8"` line will cause a style error unless all references to `python@3.8` are changed to `python@3.9` (this could be autocorrectable with `brew style --fix`, too)

This shouldn't be too hard to do. Any reason to wait until after the migration is more complete or is it better to add it now. It may depend on how many violations there are, too.

Reply:
Given this is a longer term issue, and not only with this formula, I'd suggest to move merge this for now.